storage/innobase/include/sync0policy.ic

```diff
@@ -45,6 +45,7 @@ std::string AggregateMutexStatsPolicy<Mutex>::to_string() const
 	switch (m_id) {
 
 	case LATCH_ID_BUF_BLOCK_MUTEX:
+	case LATCH_ID_BUF_POOL_ZIP:
```

Comment:
I wonder why upstream chose to use ib_bpmutex_t(AggreateStat Mutex) instead of regular ib_mutex_t.

We could use ib_mutex_t for zip mutex. There are not many of them.

At max 64 buffer pools possible. So max 64 zip mutexes.
So please try to iuse b_mutex_t for BUF_POOL_ZIP and see if that solves the problem